app/assets/javascripts/initializers/initializeSwipeGestures.js

```diff
@@ -1,61 +1,99 @@
-function initializeSwipeGestures(){
-  initializeSwipeGestures.called = true;
-  swipeState = "middle";
-  setTimeout(function(){
+/* global slideSidebar, swipeState: true */
```

Comment:
Aside from this eslint comment, the remainder of the changes in this file are all prettier code formatting.